src/sentry/tasks/store.py

```diff
@@ -771,6 +771,39 @@ def _do_save_event(
                     "events.time-to-process", time() - start_time, instance=data["platform"]
                 )
 
+            time_synthetic_monitoring_event(data, project_id, start_time)
+
+
+def time_synthetic_monitoring_event(data, project_id, start_time):
+    sm_project_id = getattr(settings, "SENTRY_SYNTHETIC_MONITORING_PROJECT_ID", None)
+    if sm_project_id is None or project_id != sm_project_id:
+        return
+
+    extra = data["extra"].get("_sentry_synthetic_monitoring")
+    if not extra:
+        return
+
+    now = time()
+    tags = {"target": extra["target"], "region": extra["region"], "source": extra["source"]}
+    instance = extra["region"]
+
+    metrics.timing(
+        "events.synthetic-monitoring.time-to-ingest-total",
+        now - data["timestamp"],
+        instance=instance,
```

Comment:
Do we really need this `instance` argument?

Reply:
I don't think so. Removing, we have enough info in tags.